src_base/xeve_eco.c

```diff
@@ -340,11 +340,71 @@ int xeve_eco_signature(XEVE_CTX * ctx, XEVE_BSW * bs)
     return XEVE_OK;
 }
 
+void write_sei_userdata_unregistered(XEVE_SEI_PAYLOAD * sei_userdata, XEVE_BSW * bs)
```

Comment:
if this function is gloabl function, "xeve_" prefix is required at the begin of function name.
But, if it is not global function, it should be declared "static" function.

Reply:
It is not a golabal function, have modified it as static function.